python-package/lightgbm/plotting.py

```diff
@@ -369,8 +402,15 @@ def plot_metric(booster, metric=None, dataset_names=None,
     return ax
 
 
-def _to_graphviz(tree_info, show_info, feature_names, precision=3,
-                 orientation='horizontal', constraints=None, **kwargs):
+def _to_graphviz(
+    tree_info: List[Dict[str, Any]],
```

Comment:
Here `tree_info` is representing one tree, not a list of trees.
```suggestion
    tree_info: Dict[str, Any],
```